src/components/v5/shared/Navigation/ColonySwitcher/ColonySwitcher.tsx

```diff
@@ -36,9 +36,11 @@ const ColonySwitcher: React.FC<ColonySwitcherProps> = ({
       <Button
         ref={setTriggerRef}
         type="button"
-        className={clsx(sidebarButtonClass, className, '!py-[6px]', {
+        className={clsx(sidebarButtonClass, className, 'group !h-[45px]', {
```

Comment:
Do we really need the height here? Is it possible to reduce the padding or adjust it somehow? I’m not a big fan of fixed sizes, but it could be useful in certain cases. Right now, it's a 50/50 situation.

https://defensivecss.dev/tip/fixed-sizes/